doc/providers/security.rst

```diff
@@ -552,18 +558,16 @@ sample users::
 Defining a custom Encoder
 ~~~~~~~~~~~~~~~~~~~~~~~~~
 
-By default, Silex uses the ``sha512`` algorithm to encode passwords.
-Additionally, the password is encoded multiple times and converted to base64.
-You can change these defaults by overriding the ``security.encoder.digest``
-service::
+By default, Silex uses the ``BCrypt`` algorithm to encode passwords.
+Additionally, the password is encoded multiple times.
+You can change these defaults by overriding the encoding cost
+``security.encoder.bcrypt.cost`` or ``security.default_encoder`` service::
 
-    use Symfony\Component\Security\Core\Encoder\MessageDigestPasswordEncoder;
+    use Symfony\Component\Security\Core\Encoder\BCryptPasswordEncoder;
 
     $app['security.encoder.digest'] = function ($app) {
```

Comment:
indeed, `security.encoder.bcrypt` or maybe `security.default_encoder` (as it could be replaced with anything)